aggregator/src/aggregator/aggregation_job_creator.rs

```diff
@@ -347,6 +380,164 @@ impl<C: Clock + 'static> AggregationJobCreator<C> {
             .await?)
     }
 
+    #[tracing::instrument(skip(self), err)]
+    async fn create_aggregation_jobs_for_fixed_size_task_no_param<
+        const L: usize,
+        A: vdaf::Aggregator<L, AggregationParam = ()>,
+    >(
+        self: Arc<Self>,
+        task: Arc<Task>,
+        max_batch_size: usize,
+    ) -> anyhow::Result<()>
+    where
+        for<'a> &'a A::AggregateShare: Into<Vec<u8>>,
+        A::PrepareMessage: Send + Sync,
+        A::PrepareState: Send + Sync + Encode,
+        A::OutputShare: Send + Sync,
+        for<'a> &'a A::OutputShare: Into<Vec<u8>>,
+    {
+        Ok(self
+            .datastore
+            .run_tx(|tx| {
+                let (this, task) = (Arc::clone(&self), Arc::clone(&task));
+                Box::pin(async move {
+                    let min_batch_size = usize::try_from(task.min_batch_size())?;
```

Comment:
Sure, makes sense. I also pedantically moved the conversions out of the transaction, on the idea that if there is a conversion problem, we're better served noticing it before we open a transaction than after. (pedantic because, on 64-bit archs, I'm pretty sure `u64` to `usize` conversion is infallible)

Reply:
Agh, unfortunately this makes the borrow-checker unhappy(!). Specifically, in the match arms, `max_batch_size` ends up being a borrow from `task`. Then trying to pass both `task` & `max_batch_size` (or `*max_batch_size`) causes the compiler to complain about task being used while borrowed. I think I could fix this by passing a `&Task` rather than an `Arc<Task>`, but the whole point of using `Arc` is that the `create_aggregation_jobs_for_fixed_size_task_no_param` needs one -- I would ultimately have to clone the `Arc`. I'd like to avoid incurring even likely-negligible runtime cost to avoid one line, so I moved the max_batch_size copy back out to `create_aggregation_jobs_for_task`.